.kiro/specs/cover-responsive-focal/design.md

```
@@ -873,91 +745,93 @@ class CRF_Render_Block_Test extends WP_UnitTestCase {
 
-#### Playwright テストスイート
+#### 重要な操作フローのテスト（20%）
+
+現在実装済みの`tests/e2e/specs/cover-block-basic.spec.ts`を基に、クリティカルパスに絞ったテストを実行：
 
 ```typescript
-// tests/e2e/cover-responsive-focal.spec.ts
-import { test, expect } from "@playwright/test";
+// tests/e2e/specs/cover-block-basic.spec.ts（現在実装済み）
+test.describe("Cover Block - Basic Responsive Focal Point Functionality", () => {
+	test("Verify plugin loads correctly", async ({ page }) => {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)